src/nrc/api/tests/test_subscriptions.py

```diff
@@ -13,13 +13,15 @@
 from nrc.datamodel.models import Domain, Subscription
 from nrc.datamodel.tests.factories import DomainFactory, SubscriptionFactory
 
+from ..scopes import SCOPE_EVENTS_CONSUMEREN
+
 
 @override_settings(
     LINK_FETCHER="vng_api_common.mocks.link_fetcher_200",
     ZDS_CLIENT_CLASS="vng_api_common.mocks.MockClient",
 )
 class SubscriptionsTestCase(JWTAuthMixin, APITestCase):
-    heeft_alle_autorisaties = True
+    scopes = [SCOPE_EVENTS_CONSUMEREN]
```

Comment:
Deze aanpassingen mag je terugzetten, ik denk dat het handig is om een aparte testcase te maken die de scopes test. Aan de huidige scopes te zien staan die nog niet helemaal goed maar dat kunnen we beter in een apart ticket oppakken.